llama_index/utilities/sql_wrapper.py

```diff
@@ -3,17 +3,9 @@
 
 from sqlalchemy import MetaData, create_engine, insert, inspect, text
 from sqlalchemy.engine import Engine
-from sqlalchemy.engine.interfaces import ReflectedIndex
 from sqlalchemy.exc import OperationalError, ProgrammingError
 
 
-def _format_index(index: ReflectedIndex) -> str:
-    return (
-        f'Name: {index["name"]}, Unique: {index["unique"]},'
-        f' Columns: {index["column_names"]!s}'
-    )
```

Comment:
was this unused?

Reply:
yes, it was previously used for the `get_table_info` function of Langchain's implementation of this class, but since we don't need that function, it was removed